tpl/cd/cd-footer/cd-social-menu.html

```diff
@@ -2,8 +2,14 @@
 
   <a href="###" class="cd-footer-social__link">
     <span class="element-invisible">Facebook</span>
-    <svg viewBox="0 0 48 48" class="facebook" aria-hidden="true"><path d="M24,5A19,19,0,1,1,5,24,19,19,0,0,1,24,5m0-3A22,22,0,1,0,46,24,22,22,0,0,0,24,2Z"/><path d="M29.16,13H25.64c-2.09,0-4.41.88-4.41,3.9,0,1.05,0,2.06,0,3.2H18.82v3.84h2.49V35h4.57V23.87h3l.27-3.78H25.81s0-1.68,0-2.17c0-1.2,1.24-1.13,1.32-1.13h2V13h0Z"/></svg>
-    <svg viewBox="0 0 48 48" class="facebook hover-style" aria-hidden="true"><path d="M24,2A22,22,0,1,0,46,24,22,22,0,0,0,24,2Zm4.9,21.87h-3V35H21.31V23.94H18.82V20.1h2.41c0-1.14,0-2.15,0-3.2,0-3,2.32-3.9,4.41-3.9h3.52v3.79h-2c-.07,0-1.32-.07-1.32,1.13,0,.49,0,2.17,0,2.17h3.37Z"/></svg>
+
+    <svg class="icon icon--facebook">
+      <use xlink:href="#sm-fb-def"></use>
+    </svg>
+
+    <svg class="icon icon--facebook hover-style">
+      <use xlink:href="#sm-fb-full"></use>
+    </svg>
```

Comment:
Nice, I liked the inlining in that it reduced requests but this is undoubtedly easier to digest when looking at it for the first time (or after a long time).